packages/features/ee/organizations/lib/server/createOrganizationFromOnboarding.ts

```diff
@@ -254,7 +255,7 @@ async function createOrMoveTeamsToOrganization(teams: TeamData[], owner: User, o
     .filter((team) => team.isBeingMigrated)
     .map((team) => ({
       id: team.id,
-      newSlug: team.slug,
+      newSlug: team.slug ? slugify(team.slug) : team.slug,
```

Comment:
This is to ensure that before we go and update that slug, it is always slugified.